Dockerfile

```diff
@@ -0,0 +1,47 @@
+#  Copyright (C) 2018-2024 LEIDOS.
+# 
+#  Licensed under the Apache License, Version 2.0 (the "License"); you may not
+#  use this file except in compliance with the License. You may obtain a copy of
+#  the License at
+# 
+#  http://www.apache.org/licenses/LICENSE-2.0
+# 
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#  License for the specific language governing permissions and limitations under
+#  the License.
+ARG DOCKER_ORG="usdotfhwastoldev"
+ARG DOCKER_TAG="develop-humble"
```

Comment:
This Dockerfile is going to be slightly awkward since we are in the middle of updating the system to ros2 humble. The DOCKER_TAG should be develop, since this code is being pushed to the develop branch for the repo.

The GIT_BRANCH however needs to still be develop-humble since the repositories being checked out are on the develop-humble branch